applications/aihorde-application.md

```diff
@@ -0,0 +1,202 @@
+with everything and easy to build around.
+
+The intended use is that 
+
+1. People with spare GPU resources will share them
+1. People with a development background will create new integrations based on
+   the AI Horde. (Frontends, Bots, Games etc)
+1. People with no GPU, or with bigger demands than their own GPU can handle
+   (e.g. text2video) will use the shared resources on the horde to utilize
+   generative AI for any purpose.
+
+### Anticipated Misuse
+
+1. For-profit or scammy services integrating into the horde and charging
+   their users for something available for free, effectively exploiting
+   the shared resources of the community for their own benefit
+1. Pedophiles using the AI Horde anonymously to generate CSAM images.
+1. Actors hating the ability for everyone to access generative AI for free,
+   and using attacks like DDOS or other vectors, to take down the service.
+
+### Countermeasures
+
+1. [The Kudos system](https://dbzer0.com/blog/the-kudos-based-economy-for-the-koboldai-horde/)
+   is built to prioritize people who are contributing back
+   to the horde. People trying to generate without contributing anything back,
+   will quickly find themselves at the back of the queue. As such, for-profit
+   leechers will quickly frustrate their users. On top of that, when we discover
+   any such services, we quickly steal their users by informing them they can
+   do the same on our free frontends, and faster.
+1. This is an area that I’ve put work-weeks of work into, to prevent and
+   discourage. I have even developed a ground-breaking technology of my own which
+   is being picked up to evaluate by researchers right now. You can read about
+   it [on my devlog](https://dbzer0.com/blog/ai-powered-anti-csam-filter-for-stable-diffusion/).
+1. This is an area that we are still potentially very vulnerable to. I have no
+   security background and I am sure I have plenty of attack vectors open. I have
+   secured things through reverse proxies and cloudflare etc. I have also developed
+   automated countermeasures to detect and block malicious actors on the worker
+   side. I also have many frontends and regular DB backups.
+
+## Needs
+
+The AI Horde has grown meterotically since I started it as a side-project to
+support my game development. I have a community in the thousands, with dozens
+of third party web services relying on the uptime of the AI Horde to serve their
+own communities! While I have a few volunteers helping me with moderation and
+development, we have no formal governance or other structure. 
+
+**Governance**: I started developing AI Horde on my own, as a side project,
+and I’m still working on it only in my spare time. Currently I am a de-facto
+“benevolent dictator”, which is not a dynamic I enjoy, and any conflicts usually
+require me to step in. As far as governance is concerned, I can at best react.
+I just don’t have the mental bandwidth to build anything, nevermind the knowledge.
```

Comment:
Knowledge we can help with! I think the best thing to do is sync with some of our open source community leaders who have done this before in the past, and learn some best practices from them on how to solve this problem. We can follow up in Discord!